internal/bminventory/inventory.go

```diff
@@ -436,7 +436,15 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 		ignitionParams["ServiceIPs"] = dataurl.EncodeBytes([]byte(ignition.GetServiceIPHostnames(b.Config.ServiceIPs)))
 	}
 
-	// [TODO] - add static network configuration only in case it was provided and this is a FullIso image
+	// [TODO] - check if this is relvant for minimal ISO also
```

Comment:
+1, perhaps for minimal ISO a different path will be required instead of `/etc/assisted/network`

Reply:
i think that it should be OK, since it is just a temp dir, and NW is not looking at it anyway, but we will check it